bedrock/etc/templates/etc/firefox/retention/thank-you-a.html

```diff
@@ -25,41 +27,45 @@
   <canvas id="confetti-canvas"></canvas>
   <header>
     <h1>
-      <span class="white">It's all</span></br>
+    {% if LANG.startswith('en') %}
+      <span class="white">It’s all</span><br/>
       thanks to you
+    {% else %}
+      {{ _('It’s all thanks to you') }}
+    {% endif %}
     </h1>
     <p>
-      Choosing Firefox helps Mozilla make the Internet<br/>a better place. Here's what you can do next:
+      {{ _('Choosing Firefox helps Mozilla make the Internet a better place. Here’s what you can do next:') }}
     </p>
   </header>
   <section>
     <article>
-      <h1>Make Firefox your default browser</h1>
-      <div class="time">1 min action</div>
-        <a href="https://support.mozilla.org/kb/make-firefox-your-default-browser">Set as your default</a>
+      <h1>{{ _('Make Firefox your default browser') }}</h1>
+      <div class="time">{{ _('1 min action') }}</div>
+        <a href="https://support.mozilla.org/kb/make-firefox-your-default-browser">{{ _('Set as your default') }}</a>
     </article>
     <article>
-      <h1>Get Firefox on your phone</h1>
-      <div class="time">2 min install</div>
-      <a href="https://support.mozilla.org/kb/install-firefox-android-device-using-google-play">Download Firefox</a>
+      <h1>{{ _('Get Firefox on your phone') }}</h1>
+      <div class="time">{{ _('2 min install') }}</div>
+      <a href="https://support.mozilla.org/kb/install-firefox-android-device-using-google-play">{{ _('Download Firefox') }}</a>
```

Comment:
Not related to this PR, but why do we send users to a SUMO article for Android here? A mozorg page like `/firefox/android/` might be better, although we have `/firefox/ios/` too. Probably worth another bug as opposed to piling on this PR.